core/stat/stat_prepare_slot.go

```diff
@@ -5,7 +5,8 @@ import (
 )
 
 const (
-	PrepareSlotName = "sentinel-core-stat-resource-node-prepare-slot"
+	PrepareSlotName  = "sentinel-core-stat-resource-node-prepare-slot"
+	PrepareSlotOrder = 1000
```

Comment:
Does the definition of a slot order, starting at 1000, have any particular meaning?

Reply:
Leave a gap for the user to add extended slots before it.

Reply:
> Leave a gap for the user to add extended slots before it.

Yes,The main thing I'm trying to say is the order of slots 1000, why isn't it 2000, or 10000.There should be a specification for it.  I don't know if there are any other better ideas for defining the slot order that Sentinel provides by default。